R/bootstrap.R

```diff
@@ -342,20 +342,13 @@ collapseSizes <- function(padding) {
 #' Create a page with a top level navigation bar
 #'
 #' Create a page that contains a top level navigation bar that can be used to
-#' toggle a set of [tabPanel()] elements.
+#' toggle a set of [tabPanel()] elements. `navbarMenu()` can be used to create
+#' an embedded menu within the navbar that in turns includes additional
+#' `tabPanels`.
```

Comment:
I've always found it a bit awkward that `navbarMenu()` is documented on the same page as `navbarPage()`, even though it can be used with any tab layout. What do you think of documenting it alongside`?tabPanel` ?

Reply:
Yeah, I think that's an improvement.